src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -1761,6 +1773,8 @@ static int rbd_nbd(int argc, const char *argv[])
 
   std::ostringstream err_msg;
   r = parse_args(args, &err_msg, &cmd, &cfg);
+  if (!err_msg.str().empty())
+    dout(1) << err_msg.str() << dendl;
```

Comment:
I don't think you may use `dout` here because the ceph context is not initialized yet. Just print it to `cerr` as it is for `r < 0` case.

Reply:
Fixed it. Thanks.